plasma_core/data/scripts/plasma_core.fnl

```diff
@@ -58,17 +58,23 @@
             using the repl if you need to add a new hook at runtime.
           An example of a hook function
                 (fn hook [self core]
-                  (core.add_hook(self, "clear_all", "On Mission About To End")
-                  (core.add_hook(self, "message_send", "On Message Received"))
+                  (core.add_hook(self :clear_all \"On Mission About To End\" ))
+                  (core.add_hook(self :message_send \"On Message Received\")))
                or in lua
                 local function hook(self, core)
-                  core.add_hook(self, "clear_all", "On Mission About To End")
-                  core.add_hook(self, "message_send", "On Message Received")
+                  core.add_hook(self, 'clear_all', 'On Mission About To End')
+                  core.add_hook(self, 'message_send', 'On Message Received')
                 end
 ")
+
+(comment requirements)
+(local reqver (require :reqver))
+(reqver:install [1 0 0])
+(local plasma_version [1 0 0])
+
 (comment General utility functions)
 
-;
+
 (fn print [output ?label]
   (comment "A safe wrapper around the engine print function, prints each on it's own line")
```

Comment:
it's -> its

Reply:
fixed, along with the corresponding lua comment.